qa/tasks/cephfs/mount.py

```diff
@@ -1189,24 +1189,30 @@ def open_n_background(self, fs_path, count):
         self.background_procs.append(rproc)
         return rproc
 
-    def create_n_files(self, fs_path, count, sync=False, dirsync=False, unlink=False, finaldirsync=False):
+    def create_n_files(self, fs_path, count, hard_links=0, sync=False,
+                       dirsync=False, unlink=False, finaldirsync=False):
```

Comment:
does this change work for all other instances of `create_n_files()`  calls ... especially since a new formal parameter has been introduced for `hard_links`

Reply:
I'll check all the instances, thanks for pointing this out

Reply:
@mchangir I checked all instances and I found `fs_path` and `count` being used in most cases as positional args while other parameters being passed as keyword args. Since there is no change in order of `fs_path` and `count`, all the instances look safe however there is an exception `self.mount_a.create_n_files("testdir/file2", 5, True)` in test_client_limits.py where the third arg which is `True` would mean `hard_links=True`, for this either I can make change to that particular line or I can shift this new arg(`hard_links`) to the end of the param list. What do you think?

Reply:
as a matter of good practice, anything new should be appended at the end as far as possible

Reply:
got it, thanks! @mchangir

Reply:
done